src/views/Main/Catalog/Components/EntityTypeList.js

```diff
@@ -60,7 +60,13 @@ export class EntityTypeList extends React.Component {
   }
 
   showError = () => {
-    this.setState({ error: true });
+    this.setState({
```

Comment:
do you mean create an object for setting the input values to empty and then use that whenever I reset them?